README.md

```diff
@@ -15,6 +15,12 @@ To install via [Composer](http://getcomposer.org/), use the command below, it wi
 composer require api-clients/psr7-oauth1 
 ```
 
+In case you want to support both PHP `7.0+` and `5.5+` in your package, we suggest you use the following command:
```

Comment:
This might be a bit misleading because `1.x` already provides PHP 7 support. So rewording this a bit to make clear that you need to support `1.x` if you have to support PHP 5.x is maybe better.